utils.py

```diff
@@ -311,6 +311,20 @@ def get_current_time_in_millisecs():
     return get_time_in_millisecs(datetime.datetime.utcnow())
 
 
+def get_current_date_as_string():
+    """Returns current date as a string of format 'YYYY-MM-DD'"""
+    return datetime.datetime.utcnow().strftime('%Y-%m-%d')
```

Comment:
Pull the format string out as a named constant that is reused in parse_date_from_string().